src/AppInstallerCLICore/Commands/SettingsCommand.cpp

```diff
@@ -79,4 +87,26 @@ namespace AppInstaller::CLI
             context << Workflow::OpenUserSetting;
         }
     }
+
+    Resource::LocString SettingsExportCommand::ShortDescription() const
+    {
+        return { Resource::String::SettingsExportCommandShortDescription };
+    }
+
+    Resource::LocString SettingsExportCommand::LongDescription() const
+    {
+        return { Resource::String::SettingsExportCommandLongDescription };
+    }
+
+    std::string SettingsExportCommand::HelpLink() const
+    {
+        return std::string{ s_SettingsCommand_HelpLink };
+    }
+
+    void SettingsExportCommand::ExecuteInternal(Execution::Context& context) const
+    {
+        context <<
+            Workflow::EnsureRunningAsAdmin <<
```

Comment:
>Workflow::EnsureRunningAsAdmin  [](http://example.com/codeflow?start=12&length=31)

Just curious is there a particular reason to require admin for export? Exporting seems harmless to me.

Reply:
There's no real reason. Removing.